CHANGELOG/CHANGELOG-1.8.md

```diff
@@ -0,0 +1,100 @@
+# Changelog 1.8
+
+## 1.8.0 - 2022-08-18
+
+### Added
+
+- Add ExternalNode feature which enables Antrea to manage the network connectivity and security on a Non-Kubernetes Node (like a virtual machine or a bare-metal server). ([#4110](https://github.com/antrea-io/antrea/pull/4110), [@wenyingd] [@mengdie-song] [@Anandkumar26])
+  * It introduces the ExternalNode CRD, each resource of which kind represents a virtual machine or bare-metal server and supports specifying which network interfaces on the external Node are expected to be protected with Antrea-native policies.
+  * An ExternalEntity resource will be created for each network interface specified in the ExternalNode resource. Antrea-native policies are applied to an external Node by using the ExternalEntity selector. 
+  * Refer to [this document](https://github.com/antrea-io/antrea/blob/release-1.8/docs/external-node.md) for more information about this feature.
+- Add the following capabilities to Antrea-native policies:
+  * Add Audit Logging support for K8s Networkpolicy. ([#4047](https://github.com/antrea-io/antrea/pull/4047), [@qiyueyao])
+  * Support applying Antrea ClusterNetworkPolicy to NodePort service for securing ingress traffic. ([#3997](https://github.com/antrea-io/antrea/pull/3997), [@GraysonWu])
+  * Introduce the Group CRD to logically group different network endpoints and reference them together in Antrea NetworkPolicy. ([#2438](https://github.com/antrea-io/antrea/pull/2438), [@qiyueyao] [@abhiraut])
+- Release Antrea Helm chart archive for each Antrea release. ([#3935](https://github.com/antrea-io/antrea/pull/3935) [#3952](https://github.com/antrea-io/antrea/pull/3952), [@antoninbas] [@yanjunz97])
```

Comment:
Release new Antrea Helm chart version for each Antrea release.

Reply:
done